C/services/notification/notification_service.cpp

```diff
@@ -133,6 +133,9 @@ bool NotificationService::start(string& coreAddress,
 		return false;
 	}
   
+	// Create the Asset Tracker
+	m_assetTracker = new AssetTracker(m_mgtClient, m_name);
```

Comment:
how the deletion of this object is taken care of ?

Reply:
Fixed

Reply:
make sure there is no crash etc. after deletion